### PR TITLE
docs: add ase doc

### PR DESCRIPTION
**Pull request by hongriTianqi**

### CODE REVIEW

File: docs/advanced/interface/ase.md [hunk 1]
```diff
@@ -1 +1,79 @@
 # ASE
+
+## Introduction
+
+[ASE](https://wiki.fysik.dtu.dk/ase/) (Atomic Simulation Environment) provides a set of Python tools for setting, running, and analysing atomic simulations. We have developed the ABACUS calculator to be used together with the ASE tools, which can be accessed from the website of [ASE](https://wiki.fysik.dtu.dk/ase/). The ABACUS calculator ([ase-abacus](https://gitlab.com/1041176461/ase-abacus )) exists as an external project with respect to ASE and is maintained by ABACUS developers.
+
+## Installation
+
+```
```
Comment (caic99) [inline docs/advanced/interface/ase.md]: You can add language mark.
```suggestion
```bash
```

**Comment by hongriTianqi** [comment]
Waiting for the upstream PR on ASE to be accepted: https://gitlab.com/ase/ase/-/merge_requests/2787

File: docs/advanced/interface/ase.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
+git clone https://gitlab.com/1041176461/ase-abacus.git
+cd ase-abacus
+python3 setup install
+```
+
+## Environment variables
+
+[ABACUS](http://abacus.ustc.edu.cn) supports two types of basis sets: PW, LCAO. The path of pseudopotential and numerical orbital files can be set throught the environment variables `ABACUS_PP_PATH` and `ABACUS_ORBITAL_PATH`, respectively, e.g.:
+
+```
```
Comment (caic99) [inline docs/advanced/interface/ase.md]:
```suggestion
```bash
```

File: docs/advanced/interface/ase.md [hunk 3]
```diff
@@ -1 +1,79 @@
...
+  export ABACUS_ORBITAL_PATH=${ORB}
+```
+ 
+For PW calculations, only `ABACUS_PP_PATH` is needed. For LCAO calculations, both `ABACUS_PP_PATH` and `ABACUS_ORBITAL_PATH` should be set.
+
+## ABACUS Calculator
+
+The default initialization command for the ABACUS calculator is
+
+```
```
Comment (caic99) [inline docs/advanced/interface/ase.md]:
```suggestion
```python
```

File: docs/advanced/interface/ase.md [hunk 4]
```diff
@@ -1 +1,79 @@
...
+|`basis_dir`     |directory where the orbital files are located, <br> Can also be specified with the `ABACUS_ORBITAL_PATH`<br> environment variable. Default: `basis_dir=./`.
+|`xc`            |which exchange-correlation functional is used.<br> An alternative way to set this parameter is via <br> seting `dft_functional` which is an ABACUS <br> parameter used to specify exchange-correlation <br> functional
+|`kpts`          |a tuple (or list) of 3 integers `kpts=(int, int, int)`, <br>it is interpreted as the dimensions of a Monkhorst-Pack <br>  grid, when `kmode` is `Gamma` or `MP`. It is <br>  interpreted as k-points, when `kmode` is `Direct`,<br>  `Cartesian` or `Line`, and `knumber` should also<br>  be set in these modes to denote the number of k-points.<br>  Some other parameters for k-grid settings:<br>  including `koffset` and `kspacing`.
+
+For more information on pseudopotentials and numerical orbitals, please visit [ABACUS]. The elaboration of input parameters can be found [here](../input_files/input-main.md).
+
+
+The input parameters can be set like::
+```
```
Comment (caic99) [inline docs/advanced/interface/ase.md]:
```suggestion
```python
```

File: docs/advanced/interface/ase.md [hunk 5]
```diff
@@ -1 +1,79 @@
...
+  calc = Abacus(profile=profile, ntype=1, ecutwfc=50, scf_nmax=50, smearing_method='gaussian', smearing_sigma=0.01, basis_type='pw', ks_solver='cg', calculation='scf' pp=pp, basis=basis, kpts=kpts)
+```
+
+The command to run jobs can be set by specifying `AbacusProfile`::
+
+```
```
Comment (caic99) [inline docs/advanced/interface/ase.md]:
```suggestion
```python
```

**Review by caic99 — APPROVED** [review]

**Pull request merged**